packages/wdio-utils/src/envDetector.ts

```diff
@@ -99,6 +100,9 @@ function isMobile(capabilities: WebdriverIO.Capabilities) {
         capabilities.platformName && capabilities.platformName.match(/ios/i) ||
         capabilities.platformName && capabilities.platformName.match(/tvos/i) ||
         capabilities.platformName && capabilities.platformName.match(/android/i) ||
+        capabilities['bstack:options']?.platformName && capabilities['bstack:options']?.platformName.match(/ios/i) ||
```

Comment:
can we also put this into a variable, e.g.:

```ts
const bsOptions = capabilities['bstack:options'] || {}
```

Reply:
Okok, I'll do it now